docs/references/api/cors.rst

```diff
@@ -1,7 +1,18 @@
+.. _cors:
+
 CORS
 ====
 
-PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
+PostgREST can set any cross origin resource sharing, that is why it can accept Ajax requests from any set of domains.
+
+You can specify the allowed CORS origins using the :code:`server-cors-allowed-origins` config. If the config is not set, it accepts requests from any domain. If the config is set to :code:`""`, it rejects requests from any domain. See :ref:`server_cors_allowed_origins`.
+
+With the below config setting, PostgREST will accept requests from :code:`http://example.com` and :code:`http://example2.com`.
+
+.. code-block::
+  
+  server-cors-allowed-origins="http://example.com, http://example2.com"
+
```

Comment:
I think we should start from the most common use case, which is accept all domains:

```suggestion
By default, PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.This behavior can be configured by using :ref:`_server_cors_allowed_origins
```

Reply:
Then at the end of this page, something like:

```
Allowed Origins
===============

With the below config setting, PostgREST will accept requests from :code:`http://example.com` and :code:`http://example2.com`.


.. code-block::
  
  server-cors-allowed-origins="http://example.com, http://example2.com"

```